### PR TITLE
[IA-4462] Address test instability

### PR DESCRIPTION
**Pull request by jdcanas**
This attempts to remediate 
```
Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. Build info: version: '4.1.1', revision: 'e8fcc2cecf' System info: host: '3f2e1c7e8264', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '3.10.0-862.3.3.el7.x86_64', java.version: '17.0.2' Driver info: org.openqa.selenium.remote.RemoteWebDriver Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--incognito, --no-experiments, --no-sandbox, --dns-prefetch-disable, --lang=en-US, --disable-setuid-sandbox, --disable-extensions, --disable-dev-shm-usage, --window-size=2880,1800, --disable-background-timer-..., --disable-backgrounding-occ..., --disable-breakpad, --disable-component-extensi..., --disable-features=Translat..., --disable-ipc-flooding-prot..., --disable-renderer-backgrou..., --enable-features=NetworkSe..., --force-color-profile=srgb, --hide-scrollbars, --metrics-recording-only, --mute-audio], extensions: [], prefs: {download.default_directory: /app/Some(/tmp), download.prompt_for_download: false, profile.block_third_party_cookies: false, profile.default_content_settings.cookies: 1}, useAutomationExtension: false}, goog:loggingPrefs: org.openqa.selenium.logging...}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--incognito, --no-experiments, --no-sandbox, --dns-prefetch-disable, --lang=en-US, --disable-setuid-sandbox, --disable-extensions, --disable-dev-shm-usage, --window-size=2880,1800, --disable-background-timer-..., --disable-backgrounding-occ..., --disable-breakpad, --disable-component-extensi..., --disable-features=Translat..., --disable-ipc-flooding-prot..., --disable-renderer-backgrou..., --enable-features=NetworkSe..., --force-color-profile=srgb, --hide-scrollbars, --metrics-recording-only, --mute-audio], extensions: [], prefs: {download.default_directory: /app/Some(/tmp), download.prompt_for_download: false, profile.block_third_party_cookies: false, profile.default_content_settings.cookies: 1}, useAutomationExtension: false}, loggingPrefs: org.openqa.selenium.logging...}}] Capabilities {}
```

### CODE REVIEW

**Review by LizBaldo — COMMENTED** [review]
Works for me, Just make sure to move this comments out of the code and into the PR itself? I am curious to hear what @rtitle thinks, but I agree that this testing should be done outside of leo (terra-docker like)

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

**Comment by jdcanas** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala [hunk 1]
```diff
@@ -287,23 +287,13 @@ trait NotebookTestUtils extends LeonardoTestUtils {
...
-          val query =
-            """! [ -d "/etc/jupyter/nbconfig" ] && echo 'true' || echo 'false' """
-
-          // Verify that /etc/jupyter/nbconfig/notebook.json exists so that welder can be enabled properly
-          val result = notebookPage.executeCell(query, timeout = 5.minutes).get
-          result should include("true")
-        }
-      }
-    else ()
+    // TODO: PR comment, we don't want to be using selenium to run notebook code as part of verifying a runtime is started, the proxy check should be sufficient
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala:296]: Why is there a TODO here? Looks like the selenium stuff was removed (so the TODO is done? :))

**Review by rtitle — APPROVED** [review]
Looks good - I agree about removing the WebDriver checks here, the Leo API validations should be sufficient.

**Pull request merged**